modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/user/application/v1/UserDiscoverableApplicationSuccessTest.java

```diff
@@ -27,13 +26,13 @@
 import org.testng.annotations.Factory;
 import org.testng.annotations.Test;
 import org.wso2.carbon.automation.engine.context.TestUserMode;
-import org.wso2.carbon.identity.application.common.model.xsd.ServiceProvider;
 
 import java.util.HashMap;
 import java.util.Map;
 import java.util.stream.IntStream;
 
 import javax.xml.xpath.XPathExpressionException;
+import org.wso2.identity.integration.test.rest.api.server.application.management.v1.model.ApplicationResponseModel;
```

Comment:
Check import order

Reply:
addressed with https://github.com/wso2/product-is/pull/23086/commits/89fe2664f35ecdd47e52e91b435d915a170577e5